run-build-functions.sh

```diff
@@ -888,3 +893,7 @@ unset_go_import_path() {
     unlink $GOPATH/src/$GO_IMPORT_PATH
   fi
 }
+
+warn() {
+  echo "WARNING: $1"
+}
```

Comment:
[sand] Some IDEs automatically insert a final newline to each file to our [`.editorconfig` file](https://github.com/netlify/build-image/blob/bac1417dd652181d67060aaf756418e6d8ca485a/.editorconfig#L11).
I.e. an unrelated PR might add a newline there.

Reply:
Note: link to setups for `.editorconfig` for different IDEs: https://editorconfig.org/#pre-installed